demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -519,29 +521,33 @@ int main(int argc, char *argv[]) {
                     faces.push_back(face);
                 }
 
+                // drawing faces
+                visualizer->draw(frame, faces);
+
                 presenter->drawGraphs(frame);
+                if (isStart) {
+                    metrics.update(startTime, frame, { 10, 22 }, cv::FONT_HERSHEY_COMPLEX,
+                        0.65, { 200, 10, 10 }, 2, PerformanceMetrics::MetricTypes::FPS);
+                    isStart = false;
+                }
+                else {
+                    metrics.update({}, frame, { 10, 22 }, cv::FONT_HERSHEY_COMPLEX,
```

Comment:
Why do G-API variants have `if-else` variant?

Reply:
This represents an inability to collect latency. https://github.com/openvinotoolkit/open_model_zoo/pull/2647#discussion_r681479296